### PR TITLE
Feature/Notify Pending Approval

### PR DESCRIPTION
**Pull request by lapv1992**
Submitter receive a message when they submit a acronym to be review.
Screen Shot 2021-07-21 at 17 41 27
_The message in the image was update, thus the :thankyou: icon wasn't show_

### CODE REVIEW

File: lambda/add_meaning.py [hunk 1]
```diff
@@ -170,13 +170,40 @@ def create_approval_request(acronym, definition, meaning, team_domain, user_id,
...
+
             response = http.request('POST', 'https://slack.com/api/chat.postMessage', body=json.dumps(modal), headers=headers)
             print("response: " + str(response.status) + " " + str(response.data))
 
         else:
             print("Skip approver due to approver sent acronym request")
 
+
+def notify_pending_approval(user_id, acronym):
+    """Sends a direct message to notify acronym is pending approvals"""
```
Comment (oijusti) [inline lambda/add_meaning.py:182]: This line might not be necessary

File: lambda/add_meaning.py [hunk 2]
```diff
@@ -170,13 +170,40 @@ def create_approval_request(acronym, definition, meaning, team_domain, user_id,
...
+    print("Sending pending approval notification...")
+    body = {
+        "channel": user_id,
+        "blocks": [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "¡Thanks for contributing! We have received your submission for '" + acronym + "'. Now it's pending approval."
```
Comment (oijusti) [inline lambda/add_meaning.py:191]: Open exclamation mark "¡"

File: lambda/add_meaning.py [hunk 3]
```diff
@@ -239,20 +266,21 @@ def lambda_handler(event, context):
     
     status_code = define(acronym,definition,meaning,notes,return_url)
     create_approval_request(acronym,definition,meaning,team_domain,user_id,user_name)
+    notify_pending_approval(user_id, acronym)
```
Comment (oijusti) [inline lambda/add_meaning.py:269]: Extra space " " before acronym

File: lambda/add_meaning.py (hunk 1 above)
Comment (lapv1992) [reply lambda/add_meaning.py]: Solved

**Review by oijusti — APPROVED** [review]

File: setup.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
     name="explain_slack_bot",
     version="0.0.1",
 
-    description="Explain is a system for definining common terms from within slack. You can also add new definitions.",
+    description="Explain is a system for defining common terms from within slack. You can also add new definitions.",
```
Comment (gihernandez) [inline setup.py:12]: Thanks!

**Review by gihernandez — APPROVED** [review]

File: lambda/add_meaning.py [hunk 5]
```diff
@@ -170,13 +170,39 @@ def create_approval_request(acronym, definition, meaning, team_domain, user_id,
...
+def notify_pending_approval(user_id, acronym):
+    print("Sending pending approval notification...")
+    body = {
+        "channel": user_id,
+        "blocks": [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "Thanks for contributing! We have received your submission for '" + acronym + "'. Now it's pending approval."
```
Comment (ne-cabrerav) [inline lambda/add_meaning.py:190]: Nitpick: you could use f-strings to format this. For example f"Thanks for contributing! We have received your submission for '{acronym}'. Now it's pending approval."

**Review by ne-cabrerav — APPROVED** [review]

File: lambda/add_meaning.py (hunk 5 above)
Comment (lapv1992) [reply lambda/add_meaning.py:190]: Will do, Thank you

**Pull request merged**